beacon-chain/cache/depositsnapshot/BUILD.bazel

```diff
@@ -19,3 +19,21 @@ go_library(
         "@com_github_pkg_errors//:go_default_library",
     ],
 )
+
+go_test(
+    name = "go_default_test",
+    srcs = [
+        "deposit_tree_snapshot_test.go",
+        "merkle_tree_test.go",
+        "spec_test.go",
+    ],
+    data = ["test_cases.yaml"],
```

Comment:
might be good to move to a testdata folder and do something like `data = glob(["testdata/**"]),`